scripts/build.js

```diff
@@ -30,6 +31,23 @@ const runWebpack = config => {
   });
 };
 
+const compileAll = async () => {
```

Comment:
Using async/await means we no longer support Node below 7.6. Might be worth sanity checking if this is a problem.